sys-cluster/drbd-utils/drbd-utils-9.12.1.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1 multilib udev
+
+LICENSE="GPL-2"
+
+DESCRIPTION="mirror/replicate block-devices across a network-connection"
+SRC_URI="http://www.linbit.com/downloads/drbd/utils/${P/_/}.tar.gz"
+HOMEPAGE="http://www.drbd.org"
```

Comment:
https://www.linbit.com/linbit-software-download-page-for-linstor-and-drbd-linux-driver/
(avoid 301)

Reply:
Changed, but preferred to use https://www.linbit.com/drbd/ as it seems to be the website containing information about the product, rather than the download listing